ci/Jenkinsfiles/build.groovy

```diff
@@ -99,7 +100,24 @@ pipeline {
                      Check formatting
                      ----------------------------------------"""
                     sh "git fetch origin 2025:origin/2025"
-                    sh "mvn -B -nsu -V -Dcustom.environment=spotless spotless:check"
+                    sh "mvn ${MAVEN_CLI_ARGS} -V -Dcustom.environment=spotless spotless:check"
+                  }
+                }
+              }
+            }
+          }
+        }
+        stage('Enforcer check') {
+          steps {
+            container('maven') {
+              warnError(message: 'Formatting check has failed') {
+                nxWithGitHubStatus(context: 'maven/enforcer', message: 'Enforce') {
```

Comment:
Should we align context on `explorer/enforcer`, or the other way around: `maven/lint`, `maven/enforcer`, like it seems to be for other addons.